website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 1
+description: Create one or more indexes on a collection to improve query performance and enable certain query features.
+---
+
+# `createIndexes`
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateIndex from '!!raw-loader!../../guides/requests/create-indexes/1-create-simple-index.js';
+import CreateUniqueIndex from '!!raw-loader!../../guides/requests/create-indexes/2-create-unique-index.js';
+import CreateCompoundIndex from '!!raw-loader!../../guides/requests/create-indexes/3-create-compound-index.js';
+import CreatePartialIndex from '!!raw-loader!../../guides/requests/create-indexes/4-create-partial-index.js';
+
+<!-- Category for Command "Administrative, Diagnostic, and Utility Commands" should be organized per folder as present in the compatibility document -->
+
+<!-- Short description of the command and its usage. -->
+
+Create one or more indexes on a given collection to improve query performance or enable certain query features (e.g., full-text, vector, and TTL indexes).
+
+<!-- Syntax structure of the command -->
+
+<!-- markdownlint-disable MD033 -->
+
+This command accepts a collection name and an array of index specification documents.
+The index specification document specifies the fields to be indexed, the index name, and other options such as uniqueness, partial filter expressions, and expiration times for TTL indexes.
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>', // string
+  indexes: [
+    {
+      key: {
+        '<field1>': 1, // ascending
+        '<field2>': -1, // descending
+        '<field3>': 'text' // for full-text search
+        // Add more fields as needed
+      },
+      name: '<index_name>', // string
+      unique: true, // Optional
+      partialFilterExpression: {
+        '<field>': { $gte: '<value>' }
+      }, // Optional
+      expireAfterSeconds: 3600, // Optional
+      sparse: true, // Optional
+      cosmosSearchOptions: {
+        kind: 'vector-diskann',
+        dimensions: 3,
+        similarity: 'COS'
+        // Add vector index options here
+      } // Optional
+    }
+    // Add more index specifications here
```

Comment:
To me, seeing syntaxes for standard indexes, partial indexes, full text search and vector search all in one place is a little busy. I've wondered if we can have syntax and table separately for some of them. Or maybe you have a better idea. :thinking: 

Having all in one place loses what's relevant, for example I don't think unique option is applicable for text search or vector search.